lib/src/bottom_sheets/cupertino_bottom_sheet.dart

```diff
@@ -24,24 +29,25 @@ const double _behind_widget_visible_height = 10;
 class _CupertinoBottomSheetContainer extends StatelessWidget {
   final Widget child;
   final Color backgroundColor;
+  final BorderRadius borderRadius;
 
   const _CupertinoBottomSheetContainer(
-      {Key key, this.child, this.backgroundColor})
+      {Key key, this.child, this.backgroundColor, this.borderRadius})
       : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     final topSafeAreaPadding = MediaQuery.of(context).padding.top;
     final topPadding = _behind_widget_visible_height + topSafeAreaPadding;
-    final radius = Radius.circular(12);
+
     final shadow =
         BoxShadow(blurRadius: 10, color: Colors.black12, spreadRadius: 5);
     final _backgroundColor =
         backgroundColor ?? CupertinoTheme.of(context).scaffoldBackgroundColor;
     return Padding(
       padding: EdgeInsets.only(top: topPadding),
       child: ClipRRect(
-        borderRadius: BorderRadius.only(topLeft: radius, topRight: radius),
+        borderRadius: borderRadius,
```

Comment:
should we add an null check here and use the _default_border_radius as fallback.

Reply:
Yess! Good catch! Also not sure about letting the user round the bottom borders. It would look really weird, I think it would be nicer to just choose the radius and not the border radius

Reply:
Yes thats right.

Reply:
So the new parameter could be `Radius topRadius`  and instead of `final radius = Radius.circular(12)` we have `final radius = topRadius ?? Radius.circular(12)`.

Reply:
Sounds good to me, I'll be updating this PR tomorrow with all your thoughts.

Reply:
Thanks!